libs/components/filter-bar/src/lib/modules/filter-bar/models/filter-bar-filter-modal-config.ts

```diff
@@ -1,9 +1,22 @@
 import { Type } from '@angular/core';
 
+import { SkyFilterBarFilterModalSizeType } from './filter-bar-filter-modal-size';
+
 export interface SkyFilterBarFilterModalConfig {
+  /**
+   * The modal component to display when the user selects the filter.
+   * The component should expect a `SkyFilterBarFilterModalContext` object to be provided by the modal service on instantiation.
+   * The return value of the modal save action should be a `SkyFilterBarFilterValue`.
```

Comment:
```suggestion
   * The return value of the modal save action is a `SkyFilterBarFilterValue`.
```
Same issue with "should" making it unclear whether this is a requirement or an option. My suggested edit just replaces the "should be" with "is," but if this is truly just optional or a recommendation, we should reword differently to make that clearer.